packages/main/cypress/specs/SplitButton.cy.tsx

```diff
@@ -0,0 +1,137 @@
+import SplitButton from "../../src/SplitButton.js";
+import Menu from "../../src/Menu.js";
+import MenuItem from "../../src/MenuItem.js";
+
+describe("Split Button general interaction", () => {
+	it("tests inner buttons design", () => {
+		cy.mount(<SplitButton id="sbDefault" design="Default">Default</SplitButton>);
+
+		cy.get("#sbDefault")
```

Comment:
select by [custom-tag]. Remove  id from the jsx template and set design different than the default one to test propagation.